runintegrationtests.sh

```diff
@@ -102,13 +102,9 @@ do
     dotnet run -p $testdir -- $TEST_PROJECT || echo "$testdir" >> $FAILURE_TEMP_FILE
   elif [[ "$COVERAGE_ARG" == "yes" && -f "$testdir/coverage.xml" ]]
   then
-    (cd $testdir; $DOTCOVER cover "coverage.xml" /ReturnTargetExitCode || echo "$testdir" >> $FAILURE_TEMP_FILE)
+    (cd $testdir; dotnet dotcover test -c Release --no-build --dcXML=coverage.xml || echo "$testdir" >> $FAILURE_TEMP_FILE)
```

Comment:
This comment applies to line 23 on this script, install_dotcover is not need anymore I think, and might generate conflicts because of different versions?

And, maybe we can now remove the instal_dotcover function from toolsversion.sh